code/script.js

```diff
@@ -1,53 +1,292 @@
+      console.log(selectedType);
+    });
+  });
+  // By pressing the Confirm button, the selectedType is confirmed
+  const animalTypeForm = document.getElementById("animal-type");
+  animalTypeForm.addEventListener("submit", (event) => {
+    event.preventDefault();
+    // Check if the user has selected a type before moving on
+    if (selectedType) {
+      showMessage(`${selectedType}, please.`, "user");
+      setTimeout(genderChoice, 1000);
+    } else {
+      showMessage(`Please select a type of ${animalChoice}.`, "bot");
+    }
+  });
+};
+
+// The user is asked to choose the gender
+const genderChoice = () => {
+  showMessage(
+    `Oh, a ${selectedType}, lovely! What gender would you like?`,
+    "bot"
+  );
+  // Gender buttons are added into the html
+  addGenderButtons();
+  // Depending on which button is clicked, the id of the clicked button is added to the gender variable
+  const genderButton = document.querySelectorAll(".send-btn");
+  console.log(genderButton);
+  genderButton.forEach((genderButton) => {
+    genderButton.addEventListener("click", (event) => {
+      gender = event.target.id;
+      console.log(gender);
+      showMessage(`A ${gender}, please.`, "user");
+      setTimeout(contactInfo, 1000);
+      switch (gender) {
+        case "male":
+          animalName = "Snufkin";
+          break;
+        case "female":
+          animalName = "Ophelia";
+          break;
+      }
+    });
+  });
+};
+
+// The user can review their choices and is asked to fill in their contact information
+const contactInfo = () => {
+  showMessage(
+    `A ${gender} ${selectedType}, you say. We have just the right ${animalChoice} for you! ${animalName} is just what you're looking for. 
+  Please enter your contact information.`,
+    "bot"
+  );
+  // Contact information form is added into the html
+  addContactForm();
+  const contactForm = document.getElementById("contact-form");
+  contactForm.addEventListener("submit", (event) => {
+    const telNo = document.getElementById("tel").value;
+    const userEmail = document.getElementById("email").value;
+    showMessage(`${telNo}, ${userEmail}`, "user");
+    event.preventDefault();
+    setTimeout(handleApplication, 1000);
+  });
+};
+
+// The user is asked to confirm their application by clicking buttons
+const handleApplication = () => {
+  showMessage(`Great! Would you like to confirm your application?`, "bot");
+  inputWrapper.innerHTML = `
+    <button class="send-btn" id="Yes">Yes</button>
+    <button class="send-btn" id="No">No</button>`;
+  document.querySelector("#Yes").addEventListener("click", () => {
+    confirmApp = "yes";
+    showMessage("Yes", "user");
+    setTimeout(appConfirmation, 1000);
+  });
+  document.querySelector("#No").addEventListener("click", () => {
+    confirmApp = "no";
+    showMessage("No", "user");
+    setTimeout(appConfirmation, 1000);
+  });
+};
+
+// The user is asked to confirm their application
+const appConfirmation = () => {
```

Comment:
To follow the naming convention, this function could be called `confirmApplication`